packages/axelar-wasm-std/src/voting.rs

```diff
@@ -222,11 +230,7 @@ impl Poll for WeightedPoll {
 
         Ok(PollResult {
             poll_id: self.poll_id,
-            results: self
-                .votes
-                .iter()
-                .map(|tally| *tally >= self.quorum.into())
-                .collect(),
+            results,
```

Comment:
in line 229 `self.status = PollStatus::Finished;` means no more votes on the partial tally